src/Processors/QueryPlan/QueryPlan.cpp

```diff
@@ -198,8 +198,6 @@ QueryPipelineBuilderPtr QueryPlan::buildQueryPipeline(
     last_pipeline->addResources(std::move(resources));
 
     /// proton : starts
-    last_pipeline->setExecuteMode(queryExecuteMode(isStreaming(), query_context->getSettingsRef()));
-
```

Comment:
already exists `setExecuteMode` in src/Interpreters/InterpreterSelectWithUnionQuery.cpp:403